doc/OnlineDocs/explanation/experimental/solvers.rst

```diff
@@ -357,3 +357,49 @@ implemented. For example, for ``ipopt``:
    :members:
    :show-inheritance:
    :inherited-members:
+
+
+Dual Sign Convention
+--------------------
+Regardless of the solver, Pyomo adopts the following sign convention. Given the problem
```

Comment:
It might be worth adding something like, "For all future solver interfaces, we have adopted the following sign convention", to make it clear that it doesn't affect anything legacy.

Reply:
Good suggestion.